test/csync/vio_tests/check_vio_ext.cpp

```diff
@@ -125,8 +101,7 @@ static int teardown(void **state) {
     rc = wipe_testdir();
     assert_int_equal(rc, 0);
 
-    SAFE_FREE(((statevar*)*state)->result);
-    SAFE_FREE(*state);
+    delete reinterpret_cast<statevar*>(*state);
```

Comment:
result doesn't need to be freed anymore?

Reply:
I now use new and delete instead of malloc